src/notation/internal/notationlivebraille.cpp

```diff
@@ -0,0 +1,292 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "io/iodevice.h"
+#include "io/buffer.h"
+
+#include "notationlivebraille.h"
+
+#include "translation.h"
+
+#include "igetscore.h"
+#include "notation.h"
+#include "internal/notationactioncontroller.h"
+
+#include "libmscore/masterscore.h"
+#include "libmscore/spanner.h"
+#include "libmscore/segment.h"
+#include "libmscore/slur.h"
+#include "libmscore/staff.h"
+#include "libmscore/part.h"
+#include "libmscore/sig.h"
+#include "libmscore/measure.h"
+
+#include "braille/internal/braille.h"
+#include "braille/internal/louis.h"
+
+#include "log.h"
+
+using namespace mu::notation;
+using namespace mu::async;
+using namespace mu::engraving;
+using namespace mu::io;
+
+NotationLiveBraille::NotationLiveBraille(const Notation* notation)
```

Comment:
I think this class better move to module braille/internal and interact with current notation via global context